apps/etterlatte-utbetaling/src/main/kotlin/utbetaling/avstemming/KonsistensavstemmingJob.kt

```diff
@@ -55,7 +55,11 @@ class KonsistensavstemmingJob(
         private val log = LoggerFactory.getLogger(this::class.java)
 
         fun run() {
-            val idag = LocalDate.now(clock.norskKlokke())
+            // val idag = LocalDate.now(clock.norskKlokke())
+            val idag =
```

Comment:
Er det noko å vinne på dette kontra å køyre ei ny avstemming i dag på vanleg vis?

Reply:
Det er faste dager som er angitt av økonomi for når vi skal sende over avstemming. 4. januar er den første i år, og den er lagt inn i kjøreplanen. Da kjører de også for denne datoen på deres side. Derfor er det ikke bare å sende over for 8. januar, for da vil det være diff mellom oss og dem.  Vi burde nok kanskje heller hatt kjøreplanen i db, slik at den kunne sjekke om den var kvittert ut der - og kjøre for datoen hvis ikke.